inst/csv/replacementPatterns.csv

```diff
@@ -401,6 +402,7 @@ impala,#,%temp_prefix%%session_id%
 impala,<hash><hash>,##
 impala,.location,.`location`
 impala,SELECT TOP @([0-9]+)rows @a;,SELECT @a LIMIT @rows;
+impala,NOT NULL,

```

Comment:
@schuemie Good point! I've updated the PR to be more specific and to add a negative test.